### PR TITLE
Add 2015 people, topics, and schedule

### PR DESCRIPTION
**Pull request by chadicus**
Lots of information is still missing, I can either keep this pull request open and add the info as it comes in, or merge this PR and open new pull requests as needed.

@chrisryan

### CODE REVIEW

File: src/config/2015.php [hunk 1]
```diff
@@ -4,18 +4,234 @@
...
+        ],
+        'legoman' => [
+            'title' => 'LegoMan',
+            'presenters' => ['brandon_beigay', 'michelle_brooks', 'chante_velez'],
+            'description' => ' Fun hands-on Lego building game (for on-site participants only) that will',
+        ],
+        'session2' => [
+            'title' => 'Refactoring',
+            'presenters' => ['chris_ryan'],
+            'description' => ' Refactoring code presentation with principals, rules, and real examples from our code bases. Examples will be mostly PHJP with some Javascript and NodeJS',
```
Comment (nubs) [inline src/config/2015.php]: PHJP?

Comment (chadicus) [inline src/config/2015.php]: say what now?

File: src/templates/schedule.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
   <ul>
     <li>Room 1 Lynnhaven <a href="http://dominionenterprises.adobeconnect.com/devcon1">http://dominionenterprises.adobeconnect.com/devcon1</a></li>
     <li>Room 2 York <a href="http://dominionenterprises.adobeconnect.com/devcon2">http://dominionenterprises.adobeconnect.com/devcon2</a></li>
+    <li>Room 3 Rappahannock <a href="http://dominionenterprises.adobeconnect.com/devcon3">http://dominionenterprises.adobeconnect.com/devcon3</a></li>
```
Comment (nubs) [inline src/templates/schedule.html]: Should Nansemond be mentioned up here too?

File: src/config/2015.php (hunk 1 above)
Comment (nubs) [inline src/config/2015.php]: Is that supposed to be `PHP` instead?

Comment (chrisryan) [inline src/config/2015.php]: Yes, that was a typo.

Comment (chadicus) [inline src/config/2015.php]: yeah, i was leaving the descriptions as-is until they presenters send me their final ones.

File: src/templates/schedule.html (hunk 2 above)
Comment (chadicus) [inline src/templates/schedule.html]: Yeah, i should. We're doing an "un-con" this year in that room so it won't be available remotely. I should add a note about that.

Comment (chrisryan) [inline src/templates/schedule.html]: The fourth room if used at all will not a media site link so I don't think it needs to be mentioned here.

**Comment by chrisryan** [comment]
You could wait till maybe Monday then go with what you have so people at least know what the schedule is.

File: src/templates/schedule.html (hunk 2 above)
Comment (nubs) [inline src/templates/schedule.html]: Cool - I wish I could be there :)

**Comment by chadicus** [comment]
Roger that.

**Comment by chadicus** [comment]
Just FYI, I removed the stuff about the forth room as it may or may not follow the same 50 minute time schedule as the normal con will.  I'm planning on making an uncon page with the schedule. I'll push to heroku throughout the day to add the sessions as they come in. or build a simple page so people can submit them.

**Comment by nubs** [comment]
That sounds like a good way to handle it :)

**Comment by chadicus** [comment]
I tried my hand at image editing. How's [this](http://cdn0.traderonline.com/v1/media/de-dev-un-con.png) for the uncon logo?

**Comment by nubs** [comment]
that actually could almost work :p

**Pull request merged**